pkgs/development/tools/riot-redis/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, stdenv
+, fetchzip
+, jre_headless
+, makeWrapper
+}:
+
+stdenv.mkDerivation rec {
+  pname = "riot-redis";
+  version = "2.17.0";
+
+  src = fetchzip {
+    url = "https://github.com/redis-developer/riot/releases/download/v${version}/${pname}-${version}.zip";
+    sha256 = "sha256-Blhjss0ypHzdd3XTRzzHnIPg8+m1ke7a+8VuBB6oA/0=";
+  };
+
+  buildInputs = [ jre_headless ];
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
```

Comment:
```suggestion
```

Reply:
i ended up having to leave this `mkdir -p $out/bin` line due to a change i made on a subsequent line. i noticed that a windows `.bat` file was in `$out/bin`. to fix this, i modified the `cp` command so that only the linux executable is copied into `$out/bin`. thanks for the suggestion though!